### PR TITLE
Remove napari prefix from plugin and module names

### PR DESCRIPTION
**Pull request by jni**
Since we can use Framework :: napari to discover, and we want to encourage more people to make napari plugins as part of bigger, non-napari-related packages, I think we should not include the napari- prefix by default. We can maybe change guidance about what should use the prefix at all, if people are feeling cautious about this.

CC @sofroniewn @tlambert03

### CODE REVIEW

**Comment by tlambert03** [comment]
I heartily agree

**Comment by sofroniewn** [comment]
>  and we want to encourage more people to make napari plugins as part of bigger, non-napari-related packages, I think we should not include the napari- prefix by default.

I feel like those people will tend to be adding napari to existing packages rather than starting new projects with the napari-cookiecutter, but i don't mind. If they do want to this now gives them that option, and it's strictly more general, so fine with me!

**Comment by sofroniewn** [comment]
Side note - do "cookiecutters" get released/ versioned? or once we merge this change will come into effect next time someone uses?

**Comment by tlambert03** [comment]
one option would be to make the "default" `plugin_name` `'napari-foobar'` ... instead of just "foobar".  (That used to not be an option because it would automatically prepend napari).  Someone could still just delete it and call it whatever they want, which satisfies the main point of this PR.  @jni?

**Comment by tlambert03** [comment]
> Side note - do "cookiecutters" get released/ versioned? or once we merge this change will come into effect next time someone uses?

the person using the `cookiecutter` command points it to a specific git repo... so if they do `cookiecutter https://github.com/napari/cookiecutter-napari-plugin` then yes, it's pointing to the current default branch (which is our master).  but one can point to anything at any point in time

**Comment by sofroniewn** [comment]
> one option would be to make the "default" plugin_name 'napari-foobar' ... instead of just "foobar". (That used to not be an option because it would automatically prepend napari).

ooo yes - I like that, it also makes it more clear for people used to the old behaviour that the behaviour has changed

**Comment by tlambert03** [comment]
see also from the [cookiecutter docs](https://cookiecutter.readthedocs.io/en/latest/usage.html#works-directly-with-git-and-hg-mercurial-repos-too) the `--checkout` option

**Comment by sofroniewn** [comment]
> see also from the cookiecutter docs the --checkout option

ah very nice, thanks for that extra info, it all sounds good!

**Comment by jni** [comment]
> I feel like those people will tend to be adding napari to existing packages rather than starting new projects

I totally get that argument, but this came from real need — I want affinder to be both a standalone package and a plugin, and this cookiecutter is just so nice, I really wanted to use it to get started! :blush: 

Anyway, I've updated the default plugin name. :tada:

**Comment by jni** [comment]
Thanks @tlambert03 for fixing the tests! Oops! :blush: 

Man, so nice to have CI finish so fast. :joy:

**Comment by tlambert03** [comment]
looks good to me! merge?

**Review by tlambert03 — APPROVED** [review]

**Comment by jni** [comment]
Boom! =)

File: {{cookiecutter.plugin_name}}/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# napari-{{cookiecutter.plugin_name}}
+# {{cookiecutter.plugin_name}}
 
-[![License](https://img.shields.io/pypi/l/napari-{{cookiecutter.plugin_name}}.svg?color=green)](https://github.com/napari/napari-{{cookiecutter.plugin_name}}/raw/master/LICENSE)
-[![PyPI](https://img.shields.io/pypi/v/napari-{{cookiecutter.plugin_name}}.svg?color=green)](https://pypi.org/project/napari-{{cookiecutter.plugin_name}})
-[![Python Version](https://img.shields.io/pypi/pyversions/napari-{{cookiecutter.plugin_name}}.svg?color=green)](https://python.org)
-[![tests](https://github.com/{{cookiecutter.github_username}}/napari-{{cookiecutter.plugin_name}}/workflows/tests/badge.svg)](https://github.com/{{cookiecutter.github_username}}/napari-{{cookiecutter.plugin_name}}/actions)
-[![codecov](https://codecov.io/gh/{{cookiecutter.github_username}}/napari-{{cookiecutter.plugin_name}}/branch/master/graph/badge.svg)](https://codecov.io/gh/{{cookiecutter.github_username}}/napari-{{cookiecutter.plugin_name}})
+[![License](https://img.shields.io/pypi/l/{{cookiecutter.plugin_name}}.svg?color=green)](https://github.com/napari/{{cookiecutter.plugin_name}}/raw/master/LICENSE)
```
Comment (ziyangczi) [inline {{cookiecutter.plugin_name}}/README.md:3]: just fyi one minor issue: this is hardcoded to napari org, and we probably want to use either github username or ask for the organization. If you want I can create a ticket for this

Comment (tlambert03) [reply {{cookiecutter.plugin_name}}/README.md:3]: can you submit a PR for it please?  change `https://github.com/napari/` to `https://github.com/napari/{{cookiecutter.github_username}}`

**Pull request merged**